forge-gui/res/cardsfolder/upcoming/saheeli_rai.txt

```diff
@@ -0,0 +1,9 @@
+Name:Saheeli Rai
+ManaCost:1 U R
+Types:Legendary Planeswalker Saheeli
+Loyalty:3
+A:AB$ Scry | Cost$ AddCounter<1/LOYALTY> | Planeswalker$ True | ScryNum$ 1 | SubAbility$ DBDealDamage | SpellDescription$ Scry 1. CARDNAME deals 1 damage to each opponent.
+SVar:DBDealDamage:DB$ DealDamage | Defined$ Player.Opponent | NumDmg$ 1 | AILogic$ Good
+A:AB$ CopyPermanent | Cost$ SubCounter<2/LOYALTY> | Planeswalker$ True | ValidTgts$ Creature.YouCtrl,Artifact.YouCtrl | TgtPrompt$ Select target artifact or creature you control. | AddTypes$ Artifact | PumpKeywords$ Haste | AtEOT$ Exile | SpellDescription$ Create a token that's a copy of target artifact or creature you control, except it's an artifact in addition to its other types. That token gains haste. Exile it at the beginning of the next end step.
```

Comment:
no `.` at end of TgtPrompt, please

Reply:
i didn't write this card, i only added the needed SubAbility$